src/components/CurrentRefinements.vue

```diff
@@ -69,12 +69,16 @@ export default {
       return this.refinements.length === 0;
     },
     refinements() {
-      let refinements = this.state.refinements.map(item => ({
-        type: item.type,
-        attribute: item.type === 'query' ? 'query' : item.attributeName,
-        label: item.computedLabel,
-        value: item,
-      }));
+      let refinements = this.state.refinements.map(item => {
+        const attribute = item.type === 'query' ? 'query' : item.attributeName;
```

Comment:
is `this.state.refinements` the result of [getRefinements](https://github.com/algolia/instantsearch.js/blob/develop/src/lib/utils.js#L227) ?
If yes, items with `type: "query"` now also have `attributeName: "query"`

Reply:
I'm aware, but we can't rely on a specific InstantSearch version to be installed.

Reply:
Can you explain why we can't force >= version on InstantSearchJS?